### PR TITLE
Improve resource quota concepts

### PR DESCRIPTION
**Pull request by tengqm**
Fix some inaccurate and/or out-dated content in the resource quota
concept page.

### CODE REVIEW

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Resource quotas work like this:
 - The administrator creates one `ResourceQuota` for each namespace.
```
Comment (sftim) [inline content/en/docs/concepts/policy/resource-quotas.md:28]: Optionally, remove the backticks here as well.

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -76,6 +76,10 @@ The following resource types are supported:
 | `limits.memory` | Across all pods in a non-terminal state, the sum of memory limits cannot exceed this value. |
 | `requests.cpu` | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
 | `requests.memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
+| `hugepages-<size>` | Across all pods in a non-terminal state, the number of
+huge page requests of the specified size cannot exceed this value. |
+| `cpu` | Same as `requests.cpu` |
+| `memory` | Same as `requests.memory` |
```
Comment (sftim) [inline content/en/docs/concepts/policy/resource-quotas.md:82]: Which should people use / is one form deprecated?

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 3]
```diff
@@ -119,12 +123,15 @@ In release 1.8, quota support for local ephemeral storage is added as an alpha f
 | ------------------------------- |----------------------------------------------------------- |
 | `requests.ephemeral-storage` | Across all pods in the namespace, the sum of local ephemeral storage requests cannot exceed this value. |
 | `limits.ephemeral-storage` | Across all pods in the namespace, the sum of local ephemeral storage limits cannot exceed this value. |
+| `ephemeral-storage` | Same as `requests.ephemeral-storage`. |
```
Comment (sftim) [inline content/en/docs/concepts/policy/resource-quotas.md:126]: Which should people use / is one form deprecated?

**Review by sftim — COMMENTED** [review]
Looks good to me

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:28]: Done.

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:82]: No, both are valid. There is no hint on which one is preferred.

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:126]: Same as above.

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 4]
```diff
@@ -138,33 +145,30 @@ Here is an example set of resources users may want to put under object count quo
...
 
 When using `count/*` resource quota, an object is charged against the quota if it exists in server storage.
 These types of quotas are useful to protect against exhaustion of storage resources.  For example, you may
-want to quota the number of secrets in a server given their large size.  Too many secrets in a cluster can
-actually prevent servers and controllers from starting!  You may choose to quota jobs to protect against
-a poorly configured cronjob creating too many jobs in a namespace causing a denial of service.
-
-Prior to the 1.9 release, it was possible to do generic object count quota on a limited set of resources.
-In addition, it is possible to further constrain quota for particular resources by their type.
+want to quota the number of Secrets in a server given their large size.  Too many Secrets in a cluster can
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/resource-quotas.md:154]: nit: Reword "to quota". You may want to count the number ...?

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 5]
```diff
@@ -173,7 +177,7 @@ exhausts the cluster's supply of Pod IPs.
 
 ## Quota Scopes
 
-Each quota can have an associated set of scopes.  A quota will only measure usage for a resource if it matches
+Each quota can have an associated set of `scopes`.  A quota will only measure usage for a resource if it matches
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/resource-quotas.md:180]: nit: two spaces before "A quota ..."

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-23770--kubernetes-io-master-staging.netlify.app/docs/concepts/policy/resource-quotas/

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 6]
```diff
@@ -18,17 +18,17 @@ Resource quotas are a tool for administrators to address this concern.
...
 
 - Different teams work in different namespaces.  Currently this is voluntary, but
   support for making this mandatory via ACLs is planned.
 
-- The administrator creates one `ResourceQuota` for each namespace.
+- The administrator creates one ResourceQuota for each namespace.
 
 - Users create resources (pods, services, etc.) in the namespace, and the quota system
-  tracks usage to ensure it does not exceed hard resource limits defined in a `ResourceQuota`.
+  tracks usage to ensure it does not exceed hard resource limits defined in a ResourceQuota.
```
Comment (irvifa) [inline content/en/docs/concepts/policy/resource-quotas.md:31]: Just curious on why we don’t need this backtick?

Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:31]: We have a style guide that says don't do this.

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 7]
```diff
@@ -605,13 +622,9 @@ For example:
...
-
-See a [detailed example for how to use resource quota](/docs/tasks/administer-cluster/quota-api-object/).
-
 ## {{% heading "whatsnext" %}}
 
 - See [ResourceQuota design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_resource_quota.md) for more information.
-
+- See a [detailed example for how to use resource quota](/docs/tasks/administer-cluster/quota-api-object/).
+- Read [Quota support for priority class design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-priority-resourcequota.md).
+- See [LimitedResources](https://github.com/kubernetes/kubernetes/pull/36765)
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/resource-quotas.md:630]: Is there a good reason to include a reference to a pull request?

Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:630]: Well ... good question. I think I relocated that link from the previous section.
Do you want me to remove it?

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 2 above)
Comment (kbhawkey) [reply content/en/docs/concepts/policy/resource-quotas.md:82]: This seems odd. Which one is preferred or how would readers decide which one to use?

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 3 above)
Comment (kbhawkey) [reply content/en/docs/concepts/policy/resource-quotas.md:126]: Yes, is one name an alias for the long form version of the name?

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 8]
```diff
@@ -138,33 +145,30 @@ Here is an example set of resources users may want to put under object count quo
...
 When using `count/*` resource quota, an object is charged against the quota if it exists in server storage.
 These types of quotas are useful to protect against exhaustion of storage resources.  For example, you may
-want to quota the number of secrets in a server given their large size.  Too many secrets in a cluster can
-actually prevent servers and controllers from starting!  You may choose to quota jobs to protect against
-a poorly configured cronjob creating too many jobs in a namespace causing a denial of service.
-
-Prior to the 1.9 release, it was possible to do generic object count quota on a limited set of resources.
-In addition, it is possible to further constrain quota for particular resources by their type.
+want to limit the number of Secrets in a server given their large size. Too many Secrets in a cluster can
+actually prevent servers and controllers from starting! You may choose to quota Jobs to protect against
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/resource-quotas.md:155]: nit: Do you need the exclamation point?
nit, break up the sentence:
You can set a quota for Jobs to protect against a poorly configured CronJob.
Cronjobs that create too many Jobs in a namespace can lead to a denial of service.

nit: It is possible to create generic count quota ...

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 9]
```diff
@@ -550,14 +585,16 @@ restrictions around nodes: pods from several namespaces may run on the same node
 
 ## Limit Priority Class consumption by default
 
-It may be desired that pods at a particular priority, eg. "cluster-services", should be allowed in a namespace, if and only if, a matching quota object exists.
+It may be desired that pods at a particular priority, eg. "cluster-services", 
+should be allowed in a namespace, if and only if, a matching quota object exists.
 
-With this mechanism, operators will be able to restrict usage of certain high priority classes to a limited number of namespaces and not every namespace will be able to consume these priority classes by default.
+With this mechanism, operators will be able to restrict usage of certain high
+priority classes to a limited number of namespaces and not every namespace
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/resource-quotas.md:592]: nit: This section is a bit unclear, but perhaps this section can be updated in a future PR.
nit: With this mechanism, operators are able to restrict usage ...

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 7 above)
Comment (kbhawkey) [reply content/en/docs/concepts/policy/resource-quotas.md:630]: Not sure whether the pull request is remaining as documentation or will be deleted in the future?
Can defer removing until later?

**Comment by kbhawkey** [comment]
@tengqm . Thanks for the updates. Noted a few nits/minor bits that could use more cleanup.
/approve

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 9 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:592]: OK

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 8 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:155]: Fixed as suggested.

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 7 above)
Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:630]: My general feeling is that as a development community, people tend to treat issues, PRs, docs and even code equally. They all serve communication channels. The kubernetes community is more unconstrained on this. Teams use Google docs, slack channel, Google forum extensively. Sometimes it is pretty difficult to build a clear understanding of the whole picture.

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 6 above)
Comment (kbhawkey) [reply content/en/docs/concepts/policy/resource-quotas.md:31]: I _think_ the guide was recently updated in support of formatting 
resource names as code.

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 7 above)
Comment (kbhawkey) [reply content/en/docs/concepts/policy/resource-quotas.md:630]: OK

**Pull request merged**